.gitignore

```diff
@@ -0,0 +1,4 @@
+**/node_modules/*
+**/npm-debug.log
+*.DS_Store
+/data/*
```

Comment:
yup. I've always used 'data' rather than 'db', despite it's the naming convention that Tyler used. I've no problem if we want to change it.